charts/prometheus-mysql-exporter/values.yaml

```diff
@@ -109,6 +112,7 @@ collectors: {}
 mysql:
   db: ""
   host: "localhost"
+  # host: &host "localhost"
```

Comment:
What is that comment for?

Reply:
Usually prometheus will have the ip of the mysql exporter in the instance label. But if we want to have the actual host name of the mysql in the instance label, we can achieve it using anchors. And so I've added it as a comment for anyone to make use of it.

Reply:
Can we remove this please?
Anchors are kind of discuraged from using with helm. See: https://helm.sh/docs/chart_template_guide/yaml_techniques/#yaml-anchors